index.bs

```diff
@@ -2772,13 +2772,31 @@ Determining the W3C Decision</h4>
 	If the proposal is adopted with changes other than <a href="#class-1">class 1 (markup) changes</a>,
 	then those changes <em class=rfc2119>must</em> be announced to the [=AC=]
 	and to the [=Group=] that owns the document (if any).
-
 	Additionally, when adopting a proposal with [=substantive changes=] integrated,
 	the announcement <em class="rfc2119">must</em> include rationale
-	for the substantive changes,
-	and those changes must have the [=consensus=]
-	of the subset of the [=AC=] that voted on the proposal
-	(including anyone who explicitly abstained).
+	for the substantive changes.
+
+	[=Substantive changes=] to the proposal <em class=rfc2119>may</em> only be adopted
+	if the revised proposal has [=consensus=]
```

Comment:
I *think* removing the `only` makes the sentence communicate what was intended. If there's any disagreement, some more substantial rephrasing and/or restructuring of the paragraph will be needed.
```suggestion
	[=Substantive changes=] to the proposal <em class=rfc2119>may</em> be adopted
	if the revised proposal has [=consensus=]
```